tests/integration.rs

```diff
@@ -6135,6 +6145,56 @@ mod common_parallel {
         handle_child_output(r, &output);
     }
 
+    #[test]
+    fn test_pvpanic() {
+        let jammy = UbuntuDiskConfig::new(JAMMY_IMAGE_NAME.to_string());
+        let guest = Guest::new(Box::new(jammy));
+        let api_socket = temp_api_path(&guest.tmp_dir);
+        let event_path = temp_event_monitor_path(&guest.tmp_dir);
+
+        let kernel_path = direct_kernel_boot_path();
+
+        let mut cmd = GuestCommand::new(&guest);
+        cmd.args(["--cpus", "boot=1"])
+            .args(["--memory", "size=512M"])
+            .args(["--kernel", kernel_path.to_str().unwrap()])
+            .args(["--cmdline", DIRECT_KERNEL_BOOT_CMDLINE])
+            .default_disks()
+            .args(["--net", guest.default_net_string().as_str()])
+            .args(["--pvpanic"])
+            .args(["--api-socket", &api_socket])
+            .args(["--event-monitor", format!("path={event_path}").as_str()])
+            .capture_output();
+
+        let mut child = cmd.spawn().unwrap();
+
+        let r = std::panic::catch_unwind(|| {
+            guest.wait_vm_boot(None).unwrap();
+
+            // Trigger guest a panic
+            make_guest_panic(&guest);
+
+            // Wait a while for guest
+            thread::sleep(std::time::Duration::new(5, 0));
+
+            // Ignore event check tempraly, as we need pvpanic-pci driver
```

Comment:
Why not add the panic driver to the kernel config file?

Reply:
Do you mean resources/linux-config-x86_64?
Will the vmlinux used in CI workloads build from this config?  Forgive me for not knowing this 😅

Reply:
Yes, the CI builds from that config file - there is an aarch64 equivalent too.

Reply:
Okay. I will try to fix this.
Thx 😸